### PR TITLE
Add domain support for in memory repo

### PR DESCRIPTION
**Pull request by borna-mdn**

### CODE REVIEW

File: domain/src/main/java/com/aconno/acnsensa/domain/repository/InMemoryRepository.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.aconno.acnsensa.domain.repository
+
+import com.aconno.acnsensa.domain.model.SensorType
+import com.aconno.acnsensa.domain.model.readings.Reading
+
+interface InMemoryRepository {
+
+    fun addReadingFor(reading: Reading)
```
Comment (digrec) [inline domain/src/main/java/com/aconno/acnsensa/domain/repository/InMemoryRepository.kt]: What is meaning of the "For" part of the function name?

File: domain/src/main/java/com/aconno/acnsensa/domain/model/readings/AccelerometerReading.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.aconno.acnsensa.domain.model.readings
+
+import com.aconno.acnsensa.domain.model.SensorType
+
+class AccelerometerReading(
+    timestamp: Long,
+    val accelerometerX: Number,
```
Comment (digrec) [inline domain/src/main/java/com/aconno/acnsensa/domain/model/readings/AccelerometerReading.kt]: Accelerometer is already in the class name, so x or xAxis is enough for property name.

Comment (borna-mdn) [reply domain/src/main/java/com/aconno/acnsensa/domain/model/readings/AccelerometerReading.kt]: Good point!

File: domain/src/main/java/com/aconno/acnsensa/domain/repository/InMemoryRepository.kt (hunk 1 above)
Comment (borna-mdn) [reply domain/src/main/java/com/aconno/acnsensa/domain/repository/InMemoryRepository.kt]: It's a typo

**Pull request merged**